teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
-    <h1>Curriculum Review</h1>
-    <p>Middle School Example Curriculum</p>
-    <p>Start the review by selecting a dimension. You do not need to complete all dimensions in one sitting. You’ll be able to download a dimension report for each dimension as well as a summary report at the end.</p>
-    <p>
-        <a href="#">
-            Content<br>
-            Covers core knowledge and skills in content standards<br>
-            6 criteria<br>
-            Est. time 30 min.
-        </a>
-    </p>
-    <p>
-        <a href="#">
-            Utility<br>
-            Supports effective teaching<br>
-            7 criteria<br>
-            Est. time 30 min.
-        </a>
-    </p>
-    <p>
-        <a href="#">
-            Quality<br>
-            Accurate and well presented<br>
-            5 criteria<br>
-            Est. time 30 min.
-        </a>
-    </p>
-    <p>
-        <a href="#">
-            Efficacy<br>
-            Improves financial knowledge, skills, or behaviors<br>
-            3 criteria<br>
-            Est. time 30 min.
-        </a>
-    </p>
-    <div class="block">
-        <h2>Content</h2>
-        <p>Curriculum content refers to the knowledge and skills covered by the curriculum. The content dimension is based on our analysis of existing national and select state financial education standards. An indicator is a concrete, detailed specification for each criteria. A component is an operational rule to help reviewers determine whether the curriculum meets each indicator. Some are essential and some are beneficial.</p>
-        <h3>Instructions</h3>
-        <p>Read through the scope and sequence of the curriculum, skim the lesson plans, and answer the following questions.</p>
+    <div class="block block__flush-top">
+        <h1>Curriculum Review</h1>
+        <p>Middle School Example Curriculum</p>
+        <p>Start the review by selecting a dimension. You do not need to complete all dimensions in one sitting. You’ll be able to download a dimension report for each dimension as well as a summary report at the end.</p>
+        <p>
+            <a href="#">
+                Content<br>
+                Covers core knowledge and skills in content standards<br>
+                6 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Utility<br>
+                Supports effective teaching<br>
+                7 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Quality<br>
+                Accurate and well presented<br>
+                5 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Efficacy<br>
+                Improves financial knowledge, skills, or behaviors<br>
+                3 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
```

Comment:
Is the design for this stuff still in process? It looks somewhat unfinished. Whatever it ends up looking like, this markup doesn't seem semantically correct. I think it should probably be an ordered list, probably within a `<nav>` element.

Reply:
This was just placeholder markup until the design is finished. This will be updated in another PR.